<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Cadastro_de_Item</class>
 <widget class="QWidget" name="Form_Cadastro_de_Item">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>840</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>940</width>
    <height>840</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>940</width>
    <height>840</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sitka Heading</family>
    <pointsize>25</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Itens</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Images/cadastro_item.png</normaloff>../Images/cadastro_item.png</iconset>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>920</width>
     <height>821</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>920</width>
     <height>821</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>920</width>
     <height>821</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::VLine</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-234</y>
      <width>899</width>
      <height>1051</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_cadastro_itens">
       <property name="minimumSize">
        <size>
         <width>880</width>
         <height>73</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>880</width>
         <height>73</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>36</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="text">
        <string>Cadastro de Itens</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_cod_item">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Cod. Do Item:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_cod_item">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="placeholderText">
          <string>Insira o código do Item...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_nome_do_item">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Nome Do Item:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_nome_do_item">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Insira o nome do Item...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_descricao_do_item">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Descrição:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_descricao_do_item">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Insira a descrição do Item...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_cod_barras_item">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cod. de Barras:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_cod_barras_item">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Insira o cod de Barras do Item...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_fornecedor_item">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Fornecedor:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_fornecedor_item">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Selecione o Fornecedor do Item...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_un_medida_item">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>UN de Medida:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_un_medida_item">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Selecione a Unidade de Medida...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_qtde_atual_item">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Qtde. Atual:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_qtde_atual_item">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Insira a Quantidade atual do Item...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_est_max_item">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Estoque Max.:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_est_max_item">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Insira o estoque máximo do Item...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_est_min_item">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Estoque Min.:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_est_min_item">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Insira o estoque mínimo do Item...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_ponto_ressup">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ponto de Ressup.:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_ponto_ressup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Insira o ponto de ressuprimento do Item...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_valor_medio_item">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Valor Médio:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_valor_medio_item">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Insira o valor médio do Item...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_setor_maior_saida_item">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Setor de Maior Saída:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_setor_maior_saida_item">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Insira o setor de maior saída do Item...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_fotos_item">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Fotos:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_fotos_item">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Insira fotos do Item...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_categoria_item">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Categoria:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_categoria_do_item">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Selecione a Categoria do Item...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_rotatividade_item">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Rotatividade:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_rotatividade">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Selecione a Rotatividade do Item...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_info_extra_item">
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>25</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Info Extra:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_info_extra">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sitka Heading</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="documentTitle">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Escreva aqui as informações extra sobre o item... (até 225 caracteres)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QPushButton" name="pushButton_cadastrar">
         <property name="text">
          <string>CADASTRAR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_limpar_tela">
         <property name="text">
          <string>LIMPAR TELA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sair">
         <property name="text">
          <string>SAIR</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_feedback">
       <property name="font">
        <font>
         <family>Sitka Heading</family>
         <pointsize>25</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255,0,0);</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhUppercaseOnly</set>
       </property>
       <property name="text">
        <string>CADASTRANDO ITEM...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
